ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -62,14 +62,14 @@ class UnityEnvironment(BaseEnv):
 
     # Default port that the editor listens on. If an environment executable
     # isn't specified, this port will be used.
-    DEFAULT_EDITOR_PORT = 5004
+    _DEFAULT_EDITOR_PORT = 5004
```

Comment:
I think this and BASE_ENVIRONMENT_PORT should stay public.

Reply:
I would prefer hiding what can be hidden, UnityEnvironment handles both the in editor and executable ports when `base-port` is None

Reply:
You're not really hiding it; it's still python so it's still visible.

Reply:
True, but it sends the message that it is not public. To me it means, it is not part of the API, you can use it but it is not supported. This way, when using an IDE, the user will first see the public members before this one.

Reply:
I am currently re-publifying them.

Reply:
Thanks. I understand the motivation, but these are important enough constants that we should make them as discoverable as possible, and avoid any chance of them going out of sync (which previously happened in our documentation).